app/src/main/java/com/example/jokesapp/data/JokesRepositoryImpl.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.jokesapp.data
+
+import com.example.jokesapp.domain.model.Jokes
+import com.example.jokesapp.domain.repos.JokesRepository
+
+class JokesRepositoryImpl : JokesRepository {
+    override fun getJokes(): List<Jokes> {
+        return listOf(
+            Jokes(
+                1,
+                "Chkara fi Taxi",
+                "Wahed rahi tahder m3a chauffeur taxi: 'Chofna chkara, matansa na7sba m3ana.'... repli chauffeur: 'Matkhafich, 3andi 3lah tsirwiya.' 😂"
```

Comment:
Here we can add parameters to each property to know what "1" refers to.
```suggestion
                id = 1,
                title = "Chkara fi Taxi",
                content = "Wahed rahi tahder m3a chauffeur taxi: 'Chofna chkara, matansa na7sba m3ana.'... repli chauffeur: 'Matkhafich, 3andi 3lah tsirwiya.' 😂"
```